pages/integrations/sso/azure_ad.md.erb

```diff
@@ -88,10 +92,11 @@ Then go to your [Azure Admin Console](https://portal.azure.com/) and select the
     1. Toggle _Enabled_ to _No_ and click _Save_
 1. Customise the User mappings:
     1. Expand _Mappings_, then click _Provision Azure Active Directory Users_
-    1. Click the first row of the _Attribute Mappings_ table, _userPrincipalName_ and change the _Source attribute_ to `mail` then click _OK_
-    1. Delete all remaining mappings except for the four listed in the following screenshot:
-
-    <%= image "azuread-user-mappings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the required user mappings on Azure" %>
+    1. Keep the following four mappings and any others:
```

Comment:
>     1. Keep the following four mappings and any others

I think we actually need them to delete the other mappings - currently our implementation doesn't ignore unknown attributes,

Reply:
Fixed on main

Reply:
thanks!

Reply:
Lol, I absolutely meant that to say 'remove any others' 😬😂